dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-record-correction-details.ts

```diff
@@ -14,10 +14,18 @@ export const getRecordCorrectionDetails = (feeRecords: FeeRecordEntity[]): FeeRe
       return [];
     }
 
-    const { facilityId, id: feeRecordId, exporter } = feeRecord;
+    const { id: feeRecordId, exporter } = feeRecord;
 
     return feeRecord.corrections.map((correction) => {
-      const { id: correctionId, dateRequested, requestedByUser } = correction;
+      const { id: correctionId, dateRequested } = correction;
+
+      /**
+       * maps the reasons array to display the current fee record values as old records
+       * constructs a comma seperated string if there are more than one reason
+       * the "old records" are the values before being corrected
+       */
+      const oldRecords = mapCorrectionReasonsToValues(feeRecord, correction.reasons);
```

Comment:
At some point we will want to map Correction Reasons to correct values so I think it would sense to name this more descriptively now to avoid confusion later:
```suggestion
      const oldRecords = mapCorrectionReasonsToOldValues(feeRecord, correction.reasons);
```